cypress/cypress/e2e/03_submariner_deploy.cy.js

```diff
@@ -24,27 +24,33 @@ describe('submariner - Deployment validation', {
     it('test the Submariner add-ons install', { tags: ['deployment'] }, function () {
         let clusterSetName = Cypress.env('CLUSTERSET')
         let nattPort = Cypress.env('SUBMARINER_IPSEC_NATT_PORT')
+        let source = Cypress.env('DOWNSTREAM_CATALOG_SOURCE')
+        let managed_clusters_list = Cypress.env('MANAGED_CLUSTERS')
+        
         clusterSetMethods.createClusterSet(clusterSetName) 
-        submarinerClusterSetMethods.manageClusterSet(clusterSetName)
+        submarinerClusterSetMethods.manageClusterSet(managed_clusters_list, clusterSetName)
 
         cy.get('button').contains('Save').click()
         cy.contains('Submariner add-ons', {timeout: 3000}).should('exist').click()
-        cy.get('#install-submariner').click()
+        cy.wait(500)
         cy.get('#install-submariner').click()
         cy.get('#available-clusters-input-toggle-select-multi-typeahead-typeahead').click()
         cy.get('.pf-c-select__menu-item').click({multiple: true})
         cy.get('#globalist-enable').click({ force: true })
         cy.get('button').contains('Next').click()
-        cy.get('#natt-port').type('{selectAll}'+nattPort)
-        for (let i=0; i<2; i++) {
+
+        cy.get('.pf-c-wizard__nav-list').eq(1).children().each(() => {
```

Comment:
What regarding the if condition for downstream/upstream deployment?

Reply:
Ignore this comment.
I forgot that we agreed, that this change will be done in a different patch.